README.md

```diff
@@ -154,18 +208,30 @@ Otherwise, `mapper` uses the default value of %Number%.
 From the `mapper` we then get an `enumMap` function from `mapper[@@toEnum]`. If `enumMap` is
 `undefined`, then we set `enumMap` to `mapper` and `mapper` to `undefined`.
 
-To support auto-initialization we also define two variables (both initialized to `undefined`): 
+To support auto-initialization we also define two variables (both initialized to **undefined**): 
 - `value`: Stores the result of the last explicit or automatic initialization.
 - `autoValue`: Stores the result of the last automatic initialization only.
 
 As we evaluate each _enum member_, we perform the following steps:
 
 1. Derive `key` from the _enum member_'s name.
-1. If the _enum member_ has an _Initializer_, then
-    1. Set `value` to be the result of evaluating _Initializer_.
+1. If the _enum member_ has an _AgebraicDataTypeParameterList_ (i.e., `(a, b, c)`), then:
+    1. Let `options` be a new Object.
+    1. Set `options.kind` to `"tuple"`.
```

Comment:
why not pass them as array/object directly?

Reply:
This is the declaration, not instance construction. There's no "object" to pass, only the property names or parameter list.